### PR TITLE
Implement vehiclePickupAir game command

### PR DESCRIPTION
**Pull request by AaronVanGeffen**

### CODE REVIEW

File: src/OpenLoco/src/Windows/Vehicle.cpp [hunk 1]
```diff
@@ -4350,15 +4367,26 @@ namespace OpenLoco::Ui::Windows::Vehicle
...
+                    auto flags = GameCommands::Flags::apply;
+                    success = GameCommands::doCommand(gcArgs, flags) != GameCommands::FAILURE;
                     break;
                 }
                 case TransportMode::air:
-                    success = GameCommands::do_59(head->id);
+                {
+                    GameCommands::VehiclePickupAirArgs gcArgs{};
+                    gcArgs.head = head->id;
+                    auto flags = GameCommands::Flags::apply | GameCommands::Flags::flag_3 | GameCommands::Flags::flag_6;
```
Comment (duncanspumpkin) [inline src/OpenLoco/src/Windows/Vehicle.cpp:4378]: should be just apply. Flags were incorrect before.

File: src/OpenLoco/src/Windows/Vehicle.cpp [hunk 2]
```diff
@@ -4350,15 +4367,26 @@ namespace OpenLoco::Ui::Windows::Vehicle
...
+                    success = GameCommands::doCommand(gcArgs, flags) != GameCommands::FAILURE;
                     break;
+                }
                 case TransportMode::water:
-                    success = GameCommands::do_63(head->id);
+                {
+                    GameCommands::VehiclePickupWaterArgs gcArgs{};
+                    gcArgs.head = head->id;
+                    auto flags = GameCommands::Flags::apply | GameCommands::Flags::flag_3 | GameCommands::Flags::flag_6;
```
Comment (duncanspumpkin) [inline src/OpenLoco/src/Windows/Vehicle.cpp:4386]: same here.

File: src/OpenLoco/src/GameCommands/VehiclePickupAir.cpp [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+                component.front->carComponent_sub_4AF16A();
+            }
+        }
+
+        return 0;
+    }
+
+    void vehiclePickupAir(Interop::registers& regs)
+    {
+        regs.ebx = vehiclePickupAir(EntityId(regs.di), regs.bl);
```
Comment (duncanspumpkin) [inline src/OpenLoco/src/GameCommands/VehiclePickupAir.cpp:68]: Use the args to handle the deserialisation.

**Review by duncanspumpkin — CHANGES_REQUESTED** [review]
See comments some minor changes/fixes

File: src/OpenLoco/src/GameCommands/VehiclePickupAir.cpp [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+            train.applyToComponents([](auto& component) {
+                component.tileX = -1;
+                component.invalidateSprite();
+                component.moveTo({ static_cast<int16_t>(0x8000), 0, 0 });
+            });
+
+            if (train.head->stationId != StationId::null)
+            {
+                auto* station = StationManager::get(train.head->stationId);
+                station->airportMovementOccupiedEdges &= ~(1 << train.head->airportMovementEdge);
```
Comment (AaronVanGeffen) [inline src/OpenLoco/src/GameCommands/VehiclePickupAir.cpp:45]:
```suggestion
                station->airportMovementOccupiedEdges &= ~(1ULL << train.head->airportMovementEdge);
```

**Review by duncanspumpkin — APPROVED** [review]

**Pull request merged**